modules/new_installer.py

```diff
@@ -0,0 +1,82 @@
+def download(package, config):
+    # Imports
+    from configparser import ConfigParser
+    from configparser import ExtendedInterpolation
+    import modules.cross_platform as cp
+    from urllib import request
+    import subprocess
+    import shutil
+
+    # Config
+    os_platform = config['OS.platform']
+    remote_location = config['Remote.location']
+    remote_branch = config['Remote.branch']
+    file_extension = config['Remote.file_extension']
+
+    # Downloading
+    print('--> Downloading Package File')
+    file_name = package + file_extension
```

Comment:
this are support package? or the package that client wants to download

Reply:
this is the name of the file plus the extension support, if the package you want to install is called yapi the full name with the extension will be yapi.sh or yapi.yp (in this new version)

Reply:
This is the .yp file for the package to download. It then downloads any extra files for installation (such as an archive) before running the script that doesn't need to get any resources externally.